DemoApp/StreamChat/Components/DemoChatChannelListVC.swift

```diff
@@ -199,8 +199,6 @@ final class DemoChatChannelListVC: ChatChannelListVC, EventsControllerDelegate {
 
     func eventsController(_ controller: EventsController, didReceiveEvent event: Event) {
         if let newMessageEvent = event as? MessageNewEvent {
-            // This is a DemoApp integration test to make sure there are no deadlocks when
```

Comment:
We probably can remove this now, no need to leave this here 🤔

Reply:
Done